### PR TITLE
Fix alignments

### PR DESCRIPTION
**Pull request by marilynechan**
* Flex form
* Margin between butons
* Dropdown alignment.

### CODE REVIEW

**Comment by wfortin** [comment]
:+1:

File: scss/forms/flex-form.scss [hunk 1]
```diff
@@ -2,15 +2,22 @@
   flex-flow: row wrap;
 
   display: flex;
+  margin-top: -10px;
+  margin-left: -10px;
```
Comment (pochretien) [inline scss/forms/flex-form.scss]: its a good idea to use a variable for this margin ? it seem the same margin for the class below but im not sure its for the same reason

**Comment by pochretien** [comment]
:ship: it

File: scss/forms/flex-form.scss (hunk 1 above)
Comment (marilynechan) [inline scss/forms/flex-form.scss]: Good idea, I added a variable!

**Pull request merged**